dicom-archive/dicomSummary.pl

```diff
@@ -199,11 +248,20 @@
 exit $returnVal;
 
 ######################################################################### end main ####################
-=pod 
-################################################
-Read archive metadata from database
-################################################
-=cut 
+
+=pod
+
+=head3 read_db_metadata($StudyUID)
+
+Reads the information stored in the database for a given StudyUID.
```

Comment:
I would say:

Accesses the database and gets the path of the file containing the metadata for the given StudyUID. What do you think?

Reply:
much better :). Thanks!